kuttl-test.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 testDirs:
   - ./tests/e2e/
-timeout: 150
-
+timeout: 600
```

Comment:
Maybe we can look for a better solution than increasing the timeout a lot.
Maybe, asserting the different deployments on the order they are started (one per KUTTL step).

Reply:
It was more a simple test that give enough time to investigate why the compactor doesnt start. :D